lib/sul_orcid_client/contributor_mapper.rb

```diff
@@ -68,12 +64,12 @@ def map_attributes
     end
 
     MARC_RELATOR_MAP = {
```

Comment:
This is an unrelated fix that accidentally made it into the PR. The capitalization was wrong on the orcid terms.